js/src/api/rpc/parity/parity.js

```diff
@@ -156,11 +172,21 @@ export default class Parity {
       .execute('parity_killAccount', inAddress(account), password);
   }
 
+  listAccounts (count, offset = null, blockNumber = 'latest') {
+    return this._transport
+      .execute('parity_listAccounts', count, inAddress(offset), inBlockNumber(blockNumber));
```

Comment:
We probably need to push the result through some formatter - I'm assuming it will contain addresses.